### PR TITLE
Add keyword arguments support to Activity classes

### PR DESCRIPTION
**Pull request by santiagodoldan**

### CODE REVIEW

**Comment by santiagodoldan** [comment]
Hello everyone 👋 , here a small change that will allow the usage of keyword arguments in activities

```
class MyWorkflow < Temporal::Workflow
  def execute(args)
     MyActivity.execute!(user_id: args[:user_id])

     ...
  end
end

class MyActivity < Temporal::Activity
  def execute(user_id:)
    puts user_id
  end
end
```

 I'm open to discuss any changes and please let me know if you can see any possible issues because of my changes so I can try to fix them 🙌

**Comment by santiagodoldan** [comment]
I can also introduce the same change here https://github.com/coinbase/temporal-ruby/blob/master/lib/temporal/workflow.rb#L15-L16

File: lib/temporal/activity.rb [hunk 1]
```diff
@@ -9,7 +9,14 @@ class Activity
...
       activity = new(context)
-      activity.execute(*input)
+
+      if input.last.instance_of?(Hash)
+        *args, kwargs = input
+
+        activity.execute(*args, **kwargs)
+      else
+        activity.execute(*input)
+      end
```
[the remark this answers is not in the document]
Comment (DeRauk) [reply lib/temporal/activity.rb:19]: @santiagodoldan apologies for the delay. Yes I tried to break it in irb and it looks like you're right, this should be ok.

**Review by DeRauk — APPROVED** [review]

**Comment by DeRauk** [comment]
@santiagodoldan I missed merging this back in July 😬 . Do you mind rebasing and pushing again to get a new test run? I'd be happy to merge then.

**Comment by santiagodoldan** [comment]
@DeRauk Thanks 🙌

**Comment by DeRauk** [comment]
@santiagodoldan could you take a look at the failing tests?

**Comment by santiagodoldan** [comment]
@DeRauk It should be fine now 💯

**Pull request merged**